cliquet/views/hello.py

```diff
@@ -13,6 +13,7 @@ def get_hello(request):
     data = dict(
         hello=project_name,
         version=settings['project_version'],
+        protocol_version=API_VERSION,
```

Comment:
Would you like me to rename ``API_VERSION`` in ``PYTHON_API_VERSION`` and ``HTTP_API_VERSION``? Or should we wait until the aren't the same?